packages/nextcloud/lib/src/patches/spreed/1-compatibility-17.1.json

```diff
@@ -1,7 +1,7 @@
 [
```

Comment:
Why isn't the name the same as the others?
Looks more like the oneOf workaround to me

Reply:
This is because the order changed. Now the compatibility patch is applied first so the oneOfs are gone. I only had to change the paths in the patches, the content didn't change at all.

Reply:
I think you got confused by the diff that is displayed here. Just check how the full patch looks like, then you'll understand.